client/src/project/Project.present.js

```diff
@@ -263,6 +263,58 @@ function ProjectViewHeaderOverviewDescription({ settingsReadOnly, description, s
   return null;
 }
 
+const ProjectSuggestActions = (props) => {
+  useEffect(() => {
+    props.fetchProjectSuggestions();
+  }, []); // eslint-disable-line
+
+  const datasets = props.core?.datasets;
+  const cantCommitsReadme = props.readme?.cantCommits;
+  const hasDatasets = typeof (datasets) === "object" ? datasets.length > 0 : true;
+
+  if (cantCommitsReadme > 1 && hasDatasets) return "";
+
+  const gitlabIDEUrl = props.externalUrl !== "" && props.externalUrl.includes("/gitlab/") ?
+    props.externalUrl.replace("/gitlab/", "/gitlab/-/ide/project/") : null;
+  const addReadmeUrl = `${gitlabIDEUrl}/edit/${props.core.default_branch}/-/README.md`;
+
+  const suggestionReadme = cantCommitsReadme > 1 ? null
+    : <li><p style={{ fontSize: "14px" }}>
+      <a className="mx-1" href={addReadmeUrl} target="_blank" rel="noopener noreferrer">
+        <strong>Add README.md</strong>
+      </a>
+      Use the README to explain your project to others, letting them understand what you want
+      to do and what you have already accomplished.
+    </p></li>;
+
+  const suggestionDataset = hasDatasets ? null
+    : <li><p style={{ fontSize: "14px" }}>
+      <Link className="mx-1" to={props.newDatasetUrl}>
+        <strong>Add some datasets</strong>
+      </Link>
+      Datasets let you work with data in a structured way, facilitating easier sharing.
+      You can create a new dataset with data you already have, or importing one from another
+      Renku project or from a public data repository such as
+      <a href="https://zenodo.org/" className="mx-1" target="_blank" rel="noreferrer noopener">
+        Zenodo</a> or
+      <a href="https://dataverse.harvard.edu/" className="mx-1" target="_blank" rel="noreferrer noopener">
+        Dataverse</a>.</p></li>;
+
+  return (
+    <InfoAlert timeout={0}>
+      <div className="mb-0" style={{ textAlign: "justify" }}>
+        <span><FontAwesomeIcon icon={faInfoCircle} /> </span>
+        <strong>Welcome</strong> to your new Renku project! There are no commits thus far in this project,
+        so here are some suggestions for getting started. <br/>
```

Comment:
The wording here is exactly as in the design, but after actually using the feature, I want to make a change:

```suggestion
        <strong>Welcome</strong> to your new Renku project! It looks like this project is just getting started, so here are some suggestions to help you. <br/>
```

Reply:
We have been reviewing nudging users towards dataset creation and it turns out that this does NOT fit all Renku use cases, for example:

- Omero users cannot create Renku datasets (we do not support this yet for streaming data)
- There is no dataset in projects that generate data outputs without using input files (simulation etc.) - we do not yet support workflow output asa dataset

Reply:
Thanks for that feedback; what you are saying makes sense. 

I think that is support to our approach of _suggesting_, but not requiring, that users make datasets. Once it is clear that the project is underway (has more than 4 commits), we do not show the messages anymore, since we assume that users have made an informed decision about whether or not they want to do the things we suggest.